common/test/acceptance/tests/studio/test_studio_grading.py

```diff
@@ -93,3 +94,68 @@ def test_grades_remain_consistent(self):
         self.grading_page.save()
         grades_alphabets = self.grading_page.grade_letters
         self.assertEqual(grades_alphabets, ['A', 'B', 'F'])
+
+    def test_staff_can_delete_grade_range(self):
+        """
+        Scenario: Users can delete grading ranges
+            Given I have opened a new course in Studio
+            And I am viewing the grading settings
+            When I add "1" new grade
+            And I delete a grade
+            Then I see I now have "2" grades
+        """
+        length = self.grading_page.total_number_of_grades
+        self.grading_page.click_add_grade()
+        self.assertTrue(self.grading_page.is_grade_added(length))
+        self.grading_page.save()
+        total_number_of_grades = self.grading_page.total_number_of_grades
+        self.assertEqual(total_number_of_grades, 3)
+        self.grading_page.remove_grades(1)
+        total_number_of_grades = self.grading_page.total_number_of_grades
+        self.assertEqual(total_number_of_grades, 2)
+
+    def test_staff_can_move_grading_ranges(self):
+        """
+        Scenario: Users can move grading ranges
+            Given I have opened a new course in Studio
+            And I am viewing the grading settings
+            When I move a grading section
+            Then I see that the grade range has changed
+        """
+        grade_ranges = self.grading_page.grades_range
+        self.assertIn('0-50', grade_ranges)
+        self.grading_page.drag_and_drop_grade()
+        grade_ranges = self.grading_page.grades_range
+        self.assertIn(
+            '0-3',
+            grade_ranges,
+            'expected range: 0-3, not found in grade ranges:{}'.format(grade_ranges)
+        )
+
+    def test_modify_assignment_type(self):
+        """
+        Scenario: Users can modify Assignment types
+            Given I have populated a new course in Studio
+            And I am viewing the grading settings
+            When I change assignment type "Homework" to "New Type"
+            And I press the "Save" notification button
+            And I go back to the main course page
+            Then I do see the assignment name "New Type"
+            And I do not see the assignment name "Homework"
+        """
+        self.grading_page.change_assignment_name('Homework', 'New Type')
+        self.grading_page.save()
+        course_outline_page = CourseOutlinePage(
+            self.browser,
+            self.course_info['org'],
+            self.course_info['number'],
+            self.course_info['run']
+        )
+        course_outline_page.visit()
+        subsection = course_outline_page.section('Test Section').subsection('Test Subsection')
+        modal = subsection.edit()
+        # Set new values
+        modal.policy = 'New Type'
```

Comment:
Why does this need to be directly changed here?  I thought the point of the test was to show that renaming the assignment on the grading page is reflected on the course outline page.  It looks like the relevant assertions in the lettuce test were disabled 4 years ago; is the test scenario still valid?

Reply:
@jmbowman Yes, you are right. Kashif suggested another way to implement the test and that was to set an assignment type for a subsection on course outline page, rename that assignment type from grading page and then come to outline page to check if the assignment type is updated on the subsection. But that did not help either because the assignment type does not get updated on the course outline page. So, I have removed the test from PR.
Is there anything else or should I merge the PR once all the checks are approved?